tool-testsuite/test/org/antlr/v4/test/tool/Java.g4

```diff
@@ -1140,7 +1140,7 @@ SignedInteger
 
 fragment
 Sign
-	:	[+-]
+	:	[+\-]
```

Comment:
So this is no longer possible without escaping the minus? It worked when the minus was at the end of the set list.